Failing test: [MKI][QA][Periodic] Security Solution Cypress.x-pack/test/security_solution_cypress/cypress/e2e/explore/inspect/inspect_button·cy·ts - Inspect Explore pages inspect Network page inspect Network page

A test failed on a tracked branch

```
AssertionError: Timed out retrying after 300000ms: Expected to find element: `[data-test-subj="sourcerer-trigger"]`, but never found it.
    at openSourcerer (webpack:///./tasks/sourcerer.ts:18:28)
    at selectDataView (webpack:///./tasks/sourcerer.ts:25:2)
    at Object.onLoad (webpack:///./e2e/explore/inspect/inspect_button.cy.ts:53:27)
    at Context.onLoad (webpack:///./tasks/navigation.ts:62:6)
```

First failure: [kibana-serverless-security-solution-quality-gate-explore - main](https://buildkite.com/elastic/kibana-serverless-security-solution-quality-gate-explore/builds/1603#01930d61-77bc-41f8-9837-781c40ce26e7)

<!-- kibanaCiData = {"failed-test":{"test.class":"Security Solution Cypress.x-pack/test/security_solution_cypress/cypress/e2e/explore/inspect/inspect_button·cy·ts","test.name":"Inspect Explore pages inspect Network page inspect Network page","test.failCount":17}} -->

--- a/x-pack/solutions/security/test/security_solution_cypress/cypress/e2e/explore/inspect/inspect_button.cy.ts
+++ b/x-pack/solutions/security/test/security_solution_cypress/cypress/e2e/explore/inspect/inspect_button.cy.ts
@@ -24,9 +24,14 @@ import { mockRiskEngineEnabled } from '../../../tasks/entity_analytics';
 
 const DATA_VIEW = 'auditbeat-*';
 
-// FLAKY: https://github.com/elastic/kibana/issues/199563
-// FLAKY: https://github.com/elastic/kibana/issues/178367
-describe.skip('Inspect Explore pages', { tags: ['@ess', '@serverless'] }, () => {
+// The Hosts and Users pages in this suite are still individually flaky/skipped for reasons
+// tracked by their own issues below. Their root cause has not been verified as fixed, so they
+// stay skipped here. Do not remove these without separately verifying and closing those issues.
+// FLAKY (Hosts page): https://github.com/elastic/kibana/issues/178367
+// FLAKY (Users page): https://github.com/elastic/kibana/issues/199583
+const SKIPPED_PAGES = ['Hosts', 'Users'];
+
+describe('Inspect Explore pages', { tags: ['@ess', '@serverless'] }, () => {
   beforeEach(() => {
     // illegal_argument_exception: unknown setting [index.lifecycle.name]
     cy.task('esArchiverLoad', { archiveName: 'risk_scores_new' });
@@ -44,7 +49,7 @@ describe.skip('Inspect Explore pages', { tags: ['@ess', '@serverless'] }, () => 
     /**
      * Group all tests of a page into one "it" call to improve speed
      */
-    it(`inspect ${pageName} page`, () => {
+    const testBody = () => {
       visitWithTimeRange(url, {
         visitOptions: {
           onLoad: () => {
@@ -79,6 +84,16 @@ describe.skip('Inspect Explore pages', { tags: ['@ess', '@serverless'] }, () => 
 
         closesModal();
       });
-    });
+    };
+
+    // NOTE: the CI spec load-balancer statically parses this file for literal `it`/`it.skip`
+    // calls, so this must stay an if/else rather than a `const itFn = cond ? it.skip : it`
+    // (a dynamic reference isn't recognized and would cause the whole file to be treated as
+    // fully skipped and excluded from CI runs).
+    if (SKIPPED_PAGES.includes(pageName)) {
+      it.skip(`inspect ${pageName} page`, testBody);
+    } else {
+      it(`inspect ${pageName} page`, testBody);
+    }
   });
 });

